sensor/kubernetes/listener/resources/virtualmachine/store/virtual_machine_store.go

```diff
@@ -100,7 +100,11 @@ func (s *VirtualMachineStore) OnNamespaceDeleted(namespace string) {
 func (s *VirtualMachineStore) Get(id virtualmachine.VMID) *virtualmachine.Info {
 	s.lock.RLock()
 	defer s.lock.RUnlock()
-	return s.virtualMachines[id].Copy()
+	vm := s.virtualMachines[id]
+	if vm == nil {
+		return nil
+	}
+	return vm.Copy()
```

Comment:
As far as I can see, `vm.Copy()` should work properly on `nil` value for `vm` (see [here](https://github.com/stackrox/stackrox/blob/f7ce63391aa45ff39a53890adb7500e1735cb059/sensor/common/virtualmachine/virtualmachineinfo.go#L18))
Nevertheless, an additional check is good.

Reply:
See [d1ee2fa](https://github.com/stackrox/stackrox/pull/17989/commits/d1ee2facc431105d89054232201db459b41ca7fe)